test/blackbox-tests/test-cases/pkg/install-action.t

```diff
@@ -11,8 +11,6 @@ Testing install actions
   $ build_pkg test
   foobar
 
-  $ export BUILD_PATH_PREFIX_MAP="/PKG_ROOT=test/target:$BUILD_PATH_PREFIX_MAP"
```

Comment:
@rgrinberg Do you know if this line is still useful somehow? The previous substitutions would only consider absolute paths, so `test/target` was ignored. The new code just replaces any path anywhere, so it introduces a change on line 38 below. Should I rather ignore relative path in `BUILD_PATH_PREFIX_MAP` to preserve the old behavior?

Reply:
The change is fine, but it needs to be behind a version check. I'd hate to break existing builds for this.

Reply:
Rebased and added a version check for the change of semantics